updates varcon.txt, en-AU

btitze opened the pull request:
pull request for updated varcon.txt

kevina commented:
@btitze Thanks, it looks like you made some changes beyond adding "D" tags, for example:
```
 # hyperesthetic (level 80)
-A: hyperesthetic / B: hyperaesthetic
-A: hyperestheticer / B: hyperaestheticer
-A: hyperestheticest / B: hyperaestheticest
+Av: hyperesthetic / A B: hyperaesthetic
+Av: hyperestheticer / A B: hyperaestheticer
+Av: hyperestheticest / A B: hyperaestheticest
```
Was this intentional?

btitze commented:
@kevina This should be
A Dv: hyperesthetic / B D: hyperaesthetic 
and so on...
Perhaps I got confused because "aesthetic" is the preferred American variant (M-W), but adding a prefix apparently changes the preference to "e" only, see "hyperesthesia" (http://www.merriam-webster.com/dictionary/hyperesthesia)

kevina commented:
Thanks a lot I found a few additional errors which I manually cleaned up in #156.

I also found a number of entities where non-D tags where modified.  Some of them corrected obvious oversights (thank you), but I am not sure about these:

```
 # barbecue <verified> (level 35)
-A B C: barbecue / AV Cv: barbeque
-A B C: barbecued / AV Cv: barbequed
-A B C: barbecuing / AV Cv: barbequing
-A B C: barbecues / AV Cv: barbeques
-A B C: barbecue's / AV Cv: barbeque's
+A B C: barbecue / AV Bv Cv: barbeque
+A B C: barbecued / AV Bv Cv: barbequed
+A B C: barbecuing / AV Bv Cv: barbequing
+A B C: barbecues / AV Bv Cv: barbeques
+A B C: barbecue's / AV Bv Cv: barbeque's
 
 # cyclopedic (level 80)
-A: cyclopedic / B: cyclopaedic
+A B: cyclopedic / Av Bv: cyclopaedic
 
 # cyclopedically (level 95)
-A: cyclopedically / B: cyclopaedically
+A B: cyclopedically / Av Bv: cyclopaedically
 
 # cyclopedist (level 80)
-A: cyclopedist / B: cyclopaedist
-A: cyclopedist's / B: cyclopaedist's
+A B: cyclopedist / Av Bv: cyclopaedist
+A B: cyclopedist's / Av Bv: cyclopaedist's
 
 # decigram (level 70)
-A C: decigram / B: decigramme
-A C: decigrams / B: decigrammes
-A C: decigram's / B: decigramme's
+A C B: decigram / Bv: decigramme
+A C B: decigrams / Bv: decigrammes
+A C B: decigram's / Bv: decigramme's
 
 # dekagram (level 70)
-A C: dekagram / B: dekagramme
-A C: dekagrams / B: dekagrammes
-A C: dekagram's / B: dekagramme's
+A B C: dekagram / Bv: dekagramme
+A B C: dekagrams / Bv: dekagrammes
+A B C: dekagram's / Bv: dekagramme's
 
 # electrohomeopathy (level 95)
-A: electrohomeopathy / B: electrohomoeopathy
-A: electrohomeopathies / B: electrohomoeopathies
-A: electrohomeopathy's / B: electrohomoeopathy's
+A B: electrohomeopathy / Bv: electrohomoeopathy
+A B: electrohomeopathies / Bv: electrohomoeopathies
+A B: electrohomeopathy's / Bv: electrohomoeopathy's
 
 # fetal <verified> (level 35)
-A: fetal / B: foetal
-A: fetaler / B: foetaler
-A: fetalest / B: foetalest
+A B C Z: fetal / Bv ZV: foetal
 
 # fetalism (level 95)
-A: fetalism / B: foetalism
+A B C Z: fetalism / Bv ZV: foetalism
 
 # fetation (level 70)
-A: fetation / B: foetation
-A: fetation's / B: foetation's
+A B C Z: fetation / Bv ZV: foetation
+A B C Z: fetation's / Bv ZV: foetation's
 
 # feticidal (level 80)
-A: feticidal / B: foeticidal
+A B C Z: feticidal / Bv ZV: foeticidal
 
 # feticide (level 70)
-A: feticide / B: foeticide
-A: feticides / B: foeticides
-A: feticide's / B: foeticide's
+A B C Z: feticide / Bv ZV: foeticide
+A B C Z: feticides / Bv ZV: foeticides
+A B C Z: feticide's / Bv ZV: foeticide's
 
 # flavorous (level 70)
-A: flavorous / B: flavourous
+A B: flavorous / Bx: flavourous
 
 # homeoblastic (level 95)
-A: homeoblastic / B: homoeoblastic
+A B: homeoblastic / Bv: homoeoblastic
 
 # homeobox (level 80)
-A: homeobox / B: homoeobox
+A B: homeobox / Bv: homoeobox
 
 # homeochromatic (level 95)
-A: homeochromatic / B: homoeochromatic
+A B: homeochromatic / Bv: homoeochromatic
 
 # homeochronous (level 95)
-A: homeochronous / B: homoeochronous
+A B: homeochronous / Bv: homoeochronous
 
 # homeocrystalline (level 95)
-A: homeocrystalline / B: homoeocrystalline
+A B: homeocrystalline / Bv: homoeocrystalline
 
 # homeogenic (level 95)
-A: homeogenic / B: homoeogenic
+A B: homeogenic / Bv: homoeogenic
 
 # homeogenous (level 95)
-A: homeogenous / B: homoeogenous
+A B: homeogenous / Bv: homoeogenous
 
 # homeoid (level 95)
-A: homeoid / B: homoeoid
+A B: homeoid / Bv: homoeoid
 
 # homeokinesis (level 95)
-A: homeokinesis / B: homoeokinesis
+A B: homeokinesis / Bv: homoeokinesis
 
 # homeomeric (level 80)
-A: homeomeric / B: homoeomeric
+A B: homeomeric / Bv: homoeomeric
 
 # homeomerous (level 80)
-A: homeomerous / B: homoeomerous
+A B: homeomerous / Bv: homoeomerous
 
 # homeomery (level 80)
-A: homeomery / B: homoeomery
-A: homeomeries / B: homoeomeries
+A B: homeomery / Bv: homoeomery
+A B: homeomeries / Bv: homoeomeries
 
 # homeomorph (level 80)
-A: homeomorph / B: homoeomorph
-A: homeomorphs / B: homoeomorphs
-A: homeomorph's / B: homoeomorph's
+A B: homeomorph / Bv: homoeomorph
+A B: homeomorphs / Bv: homoeomorphs
+A B: homeomorph's / Bv: homoeomorph's
 
 # homeomorphic (level 80)
-A: homeomorphic / B: homoeomorphic
+A B: homeomorphic / Bv: homoeomorphic
 
 # homeomorphous (level 80)
-A: homeomorphous / B: homoeomorphous
+A B: homeomorphous / Bv: homoeomorphous
 
 # homeomorphy (level 80)
-A: homeomorphy / B: homoeomorphy
-A: homeomorphies / B: homoeomorphies
+A B: homeomorphy / Bv: homoeomorphy
+A B: homeomorphies / Bv: homoeomorphies
 
 # homeopathically (level 70)
-A: homeopathically / B: homoeopathically
-A: homeopathicallier / B: homoeopathicallier
-A: homeopathicalliest / B: homoeopathicalliest
+A B: homeopathically / Bv: homoeopathically
+A B: homeopathicallier / Bv: homoeopathicallier
+A B: homeopathicalliest / Bv: homoeopathicalliest
 
 # homeopathician (level 95)
-A: homeopathician / B: homoeopathician
+A B: homeopathician / Bv: homoeopathician
 
 # homeopathicity (level 95)
-A: homeopathicity / B: homoeopathicity
+A B: homeopathicity / Bv: homoeopathicity
 
 # homeopathist (level 70)
-A: homeopathist / B: homoeopathist
-A: homeopathists / B: homoeopathists
-A: homeopathist's / B: homoeopathist's
+A B: homeopathist / Bv: homoeopathist
+A B: homeopathists / Bv: homoeopathists
+A B: homeopathist's / Bv: homoeopathist's
 
 # homeophony (level 95)
-A: homeophony / B: homoeophony
+A B: homeophony / Bv: homoeophony
 
 # homeoplasia (level 95)
-A: homeoplasia / B: homoeoplasia
+A B: homeoplasia / Bv: homoeoplasia
 
 # homeoplastic (level 95)
-A: homeoplastic / B: homoeoplastic
+A B: homeoplastic / Bv: homoeoplastic
 
 # homeoplasy (level 95)
-A: homeoplasy / B: homoeoplasy
+A B: homeoplasy / Bv: homoeoplasy
 
 # homeopolar (level 95)
-A: homeopolar / B: homoeopolar
+A B: homeopolar / Bv: homoeopolar
 
 # homeosis (level 80)
-A: homeosis / B: homoeosis
-A: homeoses / B: homoeoses
+A B: homeosis / Bv: homoeosis
+A B: homeoses / Bv: homoeoses
 
 # homeostasis (level 60)
-A: homeostasis / B: homoeostasis
-A: homeostases / B: homoeostases
-A: homeostasis's / B: homoeostasis's
+A B: homeostasis / Bv: homoeostasis
+A B: homeostases / Bv: homoeostases
+A B: homeostasis's / Bv: homoeostasis's
 
 # homeoteleuton (level 80)
-A: homeoteleuton / B: homoeoteleuton
-A: homeoteleutons / B: homoeoteleutons
+A B: homeoteleuton / Bv: homoeoteleuton
+A B: homeoteleutons / Bv: homoeoteleutons
 
 # homeothermal (level 80)
-A: homeothermal / B: homoeothermal
+A B: homeothermal / Bv: homoeothermal
 
 # homeothermic (level 80)
-A: homeothermic / B: homoeothermic
+A B: homeothermic / Bv: homoeothermic
 
 # homeothermous (level 80)
-A: homeothermous / B: homoeothermous
+A B: homeothermous / Bv: homoeothermous
 
 # homeotic (level 80)
-A: homeotic / B: homoeotic
+A B: homeotic / Bv: homoeotic
 
 # homeotype (level 95)
-A: homeotype / B: homoeotype
+A B: homeotype / Bv: homoeotype
 
 # homeotypic (level 95)
-A: homeotypic / B: homoeotypic
+A B: homeotypic / Bv: homoeotypic
 
 # homeotypical (level 95)
-A: homeotypical / B: homoeotypical
+A B: homeotypical / Bv: homoeotypical
 
 # homeozoic (level 95)
-A: homeozoic / B: homoeozoic
+A B: homeozoic / Bv: homoeozoic
 
 # humoral (level 70)
-A: humoral / B: humoural
+A B: humoral / B-: humoural
 
 # odorful (level 80)
-A: odorful / B: odourful
-A: odorfuler / B: odourfuler
-A: odorfulest / B: odourfulest
+A Cv DV: odorful / B C D: odourful
+A Cv DV: odorfuller / B C D: odourfuller
+A Cv DV: odorfullest / B C D: odourfullest

 # paleencephalon (level 95)
-A: paleencephalon / B: paleoencephalon
-A: paleencephalon's / B: paleoencephalon's
-A: paleoencephalon / B: palaeoencephalon
-A: paleoencephalon's / B: palaeoencephalon's
+A Dv: paleencephalon / B D: palaeencephalon
+A Dv: paleencephalon's / B D: palaeencephalon's
+A Dv: paleoencephalon / B D: palaeoencephalon
+A Dv: paleoencephalon's / B D: palaeoencephalon's
 
 # paleethnologic (level 80)
-A: paleethnologic / B: palaeethnologic
-A: paleethnologic / B: paleoethnologic
-A: paleoethnologic / B: palaeoethnologic
+A Dv: paleethnologic / B D: palaeethnologic
+A Dv: paleoethnologic / B D: palaeoethnologic
 
 # paleethnological (level 80)
-A: paleethnological / B: palaeethnological
-A: paleethnological / B: paleoethnological
-A: paleoethnological / B: palaeoethnological
+A Dv: paleethnological / B D: palaeethnological
+A Dv: paleoethnological / B D: palaeoethnological
 
 # paleethnologist (level 80)
-A: paleethnologist / B: palaeethnologist
-A: paleethnologist / B: paleoethnologist
-A: paleoethnologist / B: palaeoethnologist
+A Dv: paleethnologist / B D: palaeethnologist
+A Dv: paleoethnologist / B D: palaeoethnologist
 
 # paleethnology (level 70)
-A: paleethnology / B: palaeoethnology / Bv: paleoethnology
+A Dv: paleethnology / B D: palaeethnology
+A Dv: paleoethnology / B D: palaeoethnology
 
 # story <verified> (level 10)
 A B: story | book
 A B: stories | book
 A B: story's | book
-A Cv: story / AV B Bv C: storey | level of building
-A Bv: stories / AV B C Cv: storeys | level of building
-A Cv: story's / AV B Bv C: storey's | level of building
+A Cv DV: story / AV B C D: storey | level of building
+A Cv DV: stories / AV B C D: storeys | level of building
+A Cv DV: story's / AV B C D: storey's | level of building

+# licorice (level ??)
+A C D: licorice / B Dv: liquorice
+A C D: licorice's / B Dv: liquorice's
```
Can you clarify you intentions?  Thanks.

btitze commented:
I have reviewed these changes and think they are correct. Since B implies D if no D tags are present, I made these changes to reflect usage in both B and D.

barbecue
-> AU-preferred: barbecue, AU-variant: barbeque. Same as in GB, but "Bv" was missing, therefore I inserted "Bv".

cyclopedic, -pedically, -pedist
-> "e" preferred over "ae" in Oxford, Collins, Macquarie; changed entries accordingly.

decigram and dekagram
-> "-gramme" is not preferred in British/Australian English, only listed as variant. In "decigram" you can switch the tag order to "A B C".

all entries with "homeo-"
-> "homeo-" is preferred by Oxford, Collins, Chambers, Cambridge; "homoeo-" listed as variant only. B usage same as D, so no D tags necessary. Macquarie is inconsistent, lists homoeo- first for prefix, but homeopathy, homeostasis... preferred over homoeopathy, homoeostasis...

fetal, feticide, and related
-> "fetus" and derivations preferred by Oxford, Chambers, Collins, Cambridge, Macquarie; B usage as D; In the OED, "foetus" is listed as a variant in the text and not as a headword, therefore I chose ZV, not Zv. I removed "fetaler" and "fetalest" because I was sure they are non-words.  

flavorous
-> Oxford and Macquarie only list flavorous, "flavourous" should be marked as incorrect, like "humourous".

humoral
-> "humoural" should not be used, all dictionaries agree on that.

odorful
-> Derivation from "odor", used the same variant tagging. Not sure about "-fuller", "-fullest", but if these words exists, I think the "l" should be doubled. 

all entries with "pale-"
-> There were a few mistakes here that I corrected. The variations are "pale-" vs "palae-" and "paleo" vs "palaeo", not "pale-" vs "paleo".

story
-> Variant tags for B and C (for the meaning "level of buiding") appeared inconsistent, cleaned them up; added D tags.

licorice
-> This is a new entry. The word was missing in VarCon.

kevina commented:
> licorice
> -> This is a new entry. The word was missing in VarCon.

Thanks, what about the 'C' tag, did one of your sources tell you that licorice is the preferred Canadian spelling.

btitze commented:
Yes.
"licorice - the usual US and Canadian spelling of liquorice"
http://www.collinsdictionary.com/dictionary/english/licorice

kevina commented:
Thank you for your work.  I hope to release a new version of SCOWL with an Australian English dictionary for Hunspell by the end of the month.

kevina commented:
Note, I changed:
```
A B: flavorous / Bx: flavourous
```
to
```
A B: flavorous / B-: flavourous
```
as I could find some evidence for usage of the "flavourous" spelling.
